src/api/fetch.js

```diff
@@ -15,6 +15,7 @@ const defaults = {
  * @returns {Promise<{result: any, success: boolean, status: number}>}
  */
 export async function fetch() {
+  // TODO - Does this arguments parsing need to be so opaque?
```

Comment:
I've wondered about this as well! The original author of this code was an outside contributor who isn't around any more. I'm happy for this to be made more straightforward if possible.